hazelcast/src/test/java/com/hazelcast/test/TestCollectionUtils.java

```diff
@@ -1,37 +0,0 @@
-/*
- * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.hazelcast.test;
-
-import java.util.HashSet;
-import java.util.Set;
-
-/**
- * Collection of convenience methods not suitable for production usage due performance characteristics, but handy for tests.
- */
-public final class TestCollectionUtils {
-
-    private TestCollectionUtils() {
-    }
-
-    public static Set<Integer> setOfValuesBetween(int from, int to) {
-        HashSet<Integer> set = new HashSet<>();
```

Comment:
I like the change to `IntStream`, but I think it should still be centralised as it's non-trivial and repeated more than once.

Reply:
I think it is more readable this way. I can see the end index is exclusive. Before I had to check source code of TestCollectionUtils

Reply:
> I think it is more readable this way. I can see the end index is exclusive. Before I had to check source code of TestCollectionUtils

It's not clearer to me - I can never remember between `range` and `rangeClosed` which one I need. Maybe the solution could've been Javadoc on `setOfValuesBetween`?

Reply:
range is exlusive rangeClosed is inclusive as the name suggest. Why do you want to add more javadoc instead of a clear function name difference